python/pyspark/sql/functions.py

```diff
@@ -299,6 +302,22 @@ def countDistinct(col, *cols):
     return Column(jc)
 
 
+@since(1.5)
+def e():
+    """
+    Returns the float value that is closer than any other to e, the base of the natural
+    logarithms.
+
+    >>> e = sqlContext.createDataFrame([(1,)], ['a']).select((e()).alias('e')).collect()
```

Comment:
You can use the default `df`, also first() will be better than `collect()` in this case.